SharkCage/SharedFunctionality/ValidateBinary.cpp

```diff
@@ -186,41 +200,7 @@ std::vector<char> ValidateBinary::BytesToHexString(unsigned char const *bytes, s
 	return file_hash;
 }
 
-bool ValidateBinary::CompareHashes(const std::wstring &hash_1, const std::vector<char> &hash_2)
-{
-	std::string str_hash_2(hash_2.begin(), hash_2.end());
-
-	if (hash_1.size() < str_hash_2.size())
-	{
-		return false;
-	}
-
-	return std::equal(str_hash_2.begin(), str_hash_2.end(), hash_1.begin());
-}
-
-void ValidateBinary::CleanupHash(
-	const BCRYPT_ALG_HANDLE &algorithm,
-	const BCRYPT_HASH_HANDLE &hash_handle,
-	const PBYTE &hash_object,
-	const PUCHAR &hash)
+bool ValidateBinary::CompareHashes(const std::wstring &hash_1, const std::string &hash_2)
 {
-	if (algorithm)
-	{
-		::BCryptCloseAlgorithmProvider(algorithm, 0);
-	}
-
-	if (hash_handle)
-	{
-		::BCryptDestroyHash(hash_handle);
-	}
-
-	if (hash_object)
-	{
-		::HeapFree(GetProcessHeap(), 0, hash_object);
-	}
-
-	if (hash)
-	{
-		::HeapFree(GetProcessHeap(), 0, hash);
-	}
+	return std::equal(hash_2.begin(), hash_2.end(), hash_1.begin(), hash_1.end());
```

Comment:
The order is kind of bugging me 👼 if you don't mind switching it when changing the other points I would be grateful